dev/io.openliberty.cdi.4.0.internal.web/bnd.bnd

```diff
@@ -17,9 +14,6 @@ Bundle-Name: io.openliberty.cdi.4.0.internal.web
 Bundle-SymbolicName: io.openliberty.cdi.4.0.internal.web
 Bundle-Description: CDI 4.0 Web, version ${bVersion}
 
-Import-Package: org.jboss.weld.*;version="[5.1,7)", \
```

Comment:
This was added here: https://github.com/OpenLiberty/open-liberty/commit/31564b6bcc4955328f7118e6c513508aef496ff1#diff-70b4099a0254624a0439b86f34316cafc36693f5d0f8ae8daf6ff2a7e6959374R20 and I'm removing it now as it is no longer necessary